logbook-spring-boot-autoconfigure/src/main/java/org/zalando/logbook/autoconfigure/LogbookAutoConfiguration.java

```diff
@@ -245,12 +246,22 @@ public Strategy withoutBody() {
     @API(status = INTERNAL)
     @Bean
     @ConditionalOnMissingBean(Sink.class)
-    public Sink sink(
+    @ConditionalOnProperty(value = "logbook.sink.style", havingValue = "default", matchIfMissing = true)
```

Comment:
I'd move this to a separate PR. Until now the only way to configure Sinks is programmatically, by registering a bean. I have to think about whether we can (and should?) offer that also for other Sink implementations.

Reply:
(By *this*, I'm referring to the whole Spring Boot Auto Configuration support aspect of this PR.)

Reply:
Alright, will do